lib/Mojo/IOLoop/ReadWriteProcess.pm

```diff
@@ -46,6 +47,7 @@ has [
 ];
 
 has max_kill_attempts => 5;
+has kill_whole_group  => 0;
```

Comment:
that deserves some documentation as well, e.g. the text from commit message

Reply:
There's a section in the POD where it should be added

Reply:
I have already pushed a change to extend the POD. Did I add it to the wrong place? By the way, I've tried to be consistent with the existing "style".

Reply:
It was in the right place :)